tests/functional/z_cluster/nodes/test_disk_failures.py

```diff
@@ -236,3 +237,22 @@ def test_recovery_from_volume_deletion(
         self.sanity_helpers.create_resources(
             pvc_factory, pod_factory, bucket_factory, rgw_bucket_factory
         )
+
+    @bugzilla("2234479")
+    @vsphere_platform_required
+    @skipif_ocs_version("<4.15")
+    @pytest.mark.polarion_id("OCS-5502")
+    @skipif_external_mode
+    def test_recovery_from_volume_deletion_cli_tool(
```

Comment:
Do we want to still test both scenarios, mean without CLI Tool? As the TC test_recovery_from_volume_deletion will still get triggered as well.